predeployed/src/oracle_predeployed/oracle_upgradeable_generator.py

```diff
@@ -0,0 +1,40 @@
+'''Module for generation of Oracle predeployed smart contract'''
+from typing import Dict
+
+from predeployed_generator.upgradeable_contract_generator import UpgradeableContractGenerator
+
+from oracle_predeployed.oracle_generator import OracleGenerator
+
+
+class OracleUpgradeableGenerator(OracleGenerator):
+    '''Generates non upgradeable instance of OracleUpgradeable
+    '''
+
+    ARTIFACT_FILENAME = 'OracleUpgradeable.json'
+
+    # --------------- storage ---------------
+    # ------------ Initializable ------------
+    # 0:    _initialized, _initializing
+    # ---------- OracleUpgradeable ----------
+
+
+    INITIALIZED_SLOT = 0
+    ROLES_SLOT = 101
```

Comment:
`OracleUpgradeable` is not `AccessControl`.

Reply:
Yes need to remove ROLES_SLOT and ROLE_MEMBERS_SLOT